src/safeposix/syscalls/fs_calls.rs

```diff
@@ -968,17 +967,14 @@ impl Cage {
     pub fn dup_syscall(&self, fd: i32, start_desc: Option<i32>) -> i32 {
         let mut fdtable = self.filedescriptortable.write().unwrap();
 
+        //if a starting fd was passed, then use that as the starting point, but otherwise, use the designated minimum of STARTINGFD
         let start_fd = match start_desc {
             Some(start_desc) => start_desc,
             None => STARTINGFD,
         };
 
-        //checking whether the fd exists in the file table and is higher than the starting file descriptor
+        //checking whether the fd exists in the file table and is higher than the starting file descriptor or not
```

Comment:
The higher than the starting file descriptor or not part is no longer relevant to the current code